test/services/docker.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Docker Application Container Engine
+Documentation=http://docs.docker.com
+After=network.target docker.socket
+Requires=docker.socket
+
+[Service]
+ExecStart=/usr/bin/docker daemon -H fd:// --storage-driver=aufs
```

Comment:
Tests won't run in the devenv with `aufs`. Not necessarily an issue since `aufs` might in fact be a better choice given that's what we use on devices currently -- just a heads up.